src/main/java/com/denizenscript/clientizen/tags/ClientTagBase.java

```diff
@@ -60,5 +63,18 @@ public void register() {
         tagProcessor.registerTag(MapTag.class, "flag_map", (attribute, object) -> {
             return DenizenCore.serverFlagMap.doFlagMapTag(attribute);
         });
+        tagProcessor.registerTag(MaterialTag.class, "facing_material", (attribute, object) -> {
+            MinecraftClient client = MinecraftClient.getInstance();
+            if (client.crosshairTarget instanceof BlockHitResult blockHitResult) {
+                return new MaterialTag(client.world.getBlockState(blockHitResult.getBlockPos()));
+            }
+            return null;
+        });
+        tagProcessor.registerMechanism("modifyblock", false, MaterialTag.class, (object, mechanism, input) -> {
```

Comment:
maybe `modify_block`? I don't know if this is temporary, but it the conventions seem to be to use underscores for spaces in tags but keep commands without spaces

Reply:
It's just for testing, we should have a proper modifyblock command in the future